### ISSUE TITLE
Cordova build?

### ISSUE THREAD

**Issue by patrioticcow** [adnathan/ace#7]
Hi, 

Does Ace work with [phonegap build](https://build.phonegap.com/) ? In special for the native widget. 

Sorry, I haven't seen a forum where I can ask this

**Comment by adnathan** [comment]
Great question. I've been trying it out. It seems like it _should_ work, but I haven't yet gotten it to work.

**Comment by frederikbosch** [comment]
@patrioticcow Yes it does. Actually, I have problems with building with `cordova` See #37. In any case, this ticket can be closed.

**Comment by frederikbosch** [comment]
@adnathan You can close this one. Phonegap is supported. You might want to add in the readme.
